templates/telemetry/block-autoplay/block-autoplay.1.schema.json

```diff
@@ -0,0 +1,140 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+	"title": "block-autoplay",
+	"properties": {
+		"id": {
+			"type": "integer",
+			"minimum": 0
+		},
+		@TELEMETRY_CLIENTID_1_JSON@,
+		"branch": {
+			"type": "string",
+			"enum": [
+        "control",
+        "allow-and-notRemember",
+        "deny-and-notRemember",
+        "allow-and-remember",
+        "deny-and-remember"
+      ]
+		},
+		"type": {
```

Comment:
Since the following objects are conditional on type and mutually exclusive the schema should reflect that.  e.g. currently a type of counts isn't required to have a counters object (see oneOf https://json-schema.org/understanding-json-schema/reference/combining.html#oneof)

Reply:
OK, will do.